updating/understanding_updates/how-updates-work.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="how-updates-work"]
+= How cluster updates work
+include::_attributes/common-attributes.adoc[]
+:context: how-updates-work
+
+toc::[]
+
+The following sections describe each major aspect of the {product-title} (OCP) update process in detail. For a general overview of how updates work, see the xref:../../updating/understanding_updates/intro-to-updates.adoc#understanding-openshift-updates[Introduction to OpenShift updates].
```

Comment:
Do we use the OCP acronym? I don't see many (OCP) in the docs.

Reply:
I mostly see it used in filenames for release images and such, but not really for shorthand in the actual documentation. I can get rid of this acronym across this page if you'd like